kubernetes/client/ws_client_test.py

```diff
@@ -32,6 +36,24 @@ def test_websocket_client(self):
                 ]:
             self.assertEqual(get_websocket_url(url), ws_url)
 
+    @patch.object(websocket._http, '_can_use_sni')
+    @patch.object(websocket._http, '_open_socket')
+    @patch.object(ssl.SSLContext, 'load_verify_locations')
+    def test_use_of_correct_ca_cert(self, load_verify_locations, *ignored):
+        # arrange
+        expected = '/expected-ca.pem'
+        configuration.ssl_ca_cert = expected
+        load_verify_locations.side_effect = Exception("need not go further")
+
+        # act
+        with self.assertRaises(Exception):
```

Comment:
Yes, I understand the problem. But what i dont understand is why you return an exception as side_efect. 

Have you tried using None as side efect? 

What if the code raises another exception due to some missconfiguration? This test will silently fail...